_includes/fix_analytics_signup_form.html

```diff
@@ -351,6 +358,26 @@ <h2>
             return result;
         }
 
+        function updatePrice() {
+            const domainList = document.querySelectorAll("[data-domain-field]");
+            const dkkPriceElement = document.querySelector("[data-price-dkk]");
+            const eurPriceElement = document.querySelector("[data-price-eur]");
```

Comment:
You can select these outside the function to get a small performance boost (instead of selecting them each update)